src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -267,14 +258,17 @@ private void setupFileMap(final BagItDigest digest) {
     /**
      * Loads a bag config from path
      * @param bagConfigPath The path to the bag config yaml.
-     * @return
+     * @throws IOException if the bagConfigPath cannot be read
+     * @return the initialized {@link BagConfig}
      */
-    private BagConfig loadBagConfig(final String bagConfigPath) {
+    private BagConfig loadBagConfig(final String bagConfigPath) throws IOException {
         if (bagConfigPath == null) {
             throw new RuntimeException("The bag config path must not be null.");
         }
         final File bagConfigFile = new File(bagConfigPath);
-        return new BagConfig(bagConfigFile);
+        try (Reader bagConfigReader = Files.newBufferedReader(bagConfigFile.toPath())) {
```

Comment:
Would it make sense to create the "path" directly from the `bagConfigPath` (such as: `Path.of(bagConfigPath)`) instead of creating a `File` object first?